### DISCUSSION TITLE
Reading more than 1 file on the same time.

### DISCUSSION THREAD

**Discussion by menezesluan** [kedro-org/kedro#3011] [Idea]
Hello Team, how are you?
Context:
I  have a CloudStorage/bucket with that struct below, and I would to read the content on the same time.
 | bucket_name
 | — folder
 | — | — 001recovered.txt
 | — | — 002recovered.txt

Question:
How it’s possible to do that? I’m getting a error.

`catalog.yml` 

relvenda:
  type: text.TextDataSet
  filepath: "gs://bucket_name/folder/*recovered.txt"

**Comment by astrojuanlu** [comment]
Relaying answer from Slack https://linen-slack.kedro.org/t/15637169/hello-team-how-are-you-context-i-have-a-cloudstorage-bucket-#e5e9639d-d07f-45d3-bf1b-429720dbbd26

> You'll likely be looking for the functionality that `PartitionedDataset` offers: https://docs.kedro.org/en/stable/kedro.io.PartitionedDataset.html#kedro.io.PartitionedDataset

**Reply by menezesluan** [reply]
Thank you!! It works
